tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -705,3 +705,148 @@ def _find_output(predictions):
                 'If you need to use another layer, please specify it explicitly'
             )
         return filter_outputs[0]
+
+
+class FaceBoxesSSDDecoder(Adapter):
+    """
+    Class for converting output of FaceBoxes SSD models to DetectionPrediction representation
```

Comment:
SSD or Single Shot Detector is another topology, which do not have relation to FaceBoxes
```suggestion
    Class for converting output of FaceBoxes models to DetectionPrediction representation
```

Reply:
Fixed